docs/CHANGELOG.md

```diff
@@ -79,6 +79,10 @@ after reboot before the first transaction is ever sent, eliminating the previous
 scenario where the node could send underpriced or overpriced transactions for a
 period after a reboot, until the gas updater caught up.
 
+- Performance improvements to OCR job adds. Removed the pipeline_task_specs table
```

Comment:
From a node op's perspective, how does this improve my life?

e.g. "Fixed long delay when archiving OCR jobs. Archiving jobs should now complete within a second or so at most"

Reply:
I'm hesitant to put some specific performance guarantee on it. I just ran a little benchmark on CreateSpec and it went from `11966126 ns/op` to `1188095 ns/op` with this change improvement, but thats one component of the overall job create and there is a lot of network i/o so it could vary a lot. 

As for the table comment, I think it makes sense to give a heads up if a whole table is being deprecated but I can leave that out if you think ops don't care.